test/DynamoCoreWpfTests/ViewExtensions/PythonMigrationViewExtensionTests.cs

```diff
@@ -43,10 +33,6 @@ public void WillDisplayDialogWhenOpeningGraphWithIronPythonNodes()
         {
             DebugModes.LoadDebugModesStatusFromConfig(Path.Combine(GetTestDirectory(ExecutingDirectory), "DynamoCoreWpfTests", "python3DebugMode.config"));
             DynamoModel.IsTestMode = false;
-            // Arrange
-            RaiseLoadedEvent(this.View);
-            var extensionManager = View.viewExtensionManager;
-            extensionManager.Add(viewExtension);
```

Comment:
I guess this should be here anyway because by default Dynamo loads all viewExtension with xml file there

Reply:
yes, this was attempting to reload and causing issues.